### ISSUE TITLE
Implent --replace and --append parameters to the publish command 

### ISSUE THREAD

**Issue by veruzi** [tableau/tabcmd#252]
This is to keep tabcmd1 and tabcmd2 on par. I'm using these in tabcmd1 already for years.

**Comment by jacalata** [comment]
good catch - we do have these on our internal list of remaining gaps, but I don't have an ETA for that work. It will require a server update, not just an update here.

**Comment by jacalata** [comment]
Fixed in #356
